src/main/java/com/exactpro/th2/act/ActHandler.java

```diff
@@ -441,6 +484,27 @@ private EventID createAndStoreParentEvent(PlaceMessageRequestOrBuilder request,
         }
     }
 
+    private EventID createAndStoreParentEvent(MultiSendRequest request, String actName, Status status) throws IOException {
+        long startTime = System.currentTimeMillis();
+
+        Event event = start()
+                .name(actName + ' ' + request.getMessages(0).getMetadata().getId().getConnectionId().getSessionAlias())
+                .description(request.getDescription())
+                .type(actName)
+                .status(status)
+                .endTimestamp(); // FIXME set properly as is in the last child
+
+        com.exactpro.th2.common.grpc.Event protoEvent = event.toProto(request.getParentEventId());
+        //FIXME process response
+        try {
+            eventBatchMessageRouter.send(EventBatch.newBuilder().addEvents(event.toProto(request.getParentEventId())).build(), "publish", "event");
```

Comment:
There is no need in `publish` and `event` attributes as they are automatically added by the router